### PR TITLE
add ahr999

### PR DESCRIPTION
**Pull request by who3m1**

### CODE REVIEW

File: src/static/bots/7000103262/icon.png [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 2000 2000" width="2000" height="2000"><path d="M1970,1242c-133.48,535.66-676.18,861.68-1212,728S-103.69,1293.89,30,758.18,706.12-103.7,1241.82,30,2103.69,706.16,1970,1242h0Z" fill="#f7931a"/><path d="M1441,857.53c19.88-133.07-81.44-204.61-220-252.33l45-180.25-109.76-27.34-43.84,175.51c-28.82-7.18-58.44-14-87.88-20.68l44-176.67L958.87,348.43,914,528.61,692.68,473.79,663.48,591s81.42,18.66,79.7,19.82c44.44,11.1,52.5,40.5,51.14,63.82l-123.14,493.8c-5.44,13.5-19.22,33.74-50.28,26,1.1,1.6-79.76-20-79.76-20l-54.48,125.79,221,55.8-45.42,182.35,109.6,27.34,45-180.39c30,8.12,59,15.62,87.42,22.68l-44.82,179.55,109.74,27.34,45.42-182c187.13,35.4,327.85,21.12,387-148,47.72-136.25-2.34-214.85-100.8-266.13,71.7-16.6,125.7-63.74,140.11-161.17m-250.71,351.52c-34,136.25-263.35,62.62-337.77,44.12l60.26-241.55c74.38,18.56,312.89,55.32,277.55,197.41m34-353.5c-30.94,124-221.91,61-283.89,45.54L994.91,682c62,15.48,261.51,44.3,229.25,173.59" fill="#fff"/></svg>
```
Comment (lyricat) [inline src/static/bots/7000103262/icon.png:1]: it's a svg, not png

File: src/static/bots/7000103262/index.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{
+  "id": "7000103262",
+  "client_id": "5551a7a2-93b1-43be-afcd-8a18fee25f0d",
+  "url": "ahr999mixin.tk",
```
Comment (lyricat) [inline src/static/bots/7000103262/index.json:4]: should add protocol name, for example: https://

File: src/static/bots/7000103262/index.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  "name": {
+    "en": "Ahr999 index",
+    "zh": "Ahr999指数"
+  },
+  "text": {
+    "en": ["realtime ahr999 index"],
+    "zh": ["订阅与实时获取Ahr999指数"]
+  },
+  "category": "information",
```
Comment (lyricat) [inline src/static/bots/7000103262/index.json:13]: there is no `information` category now, but you can use `util` instead.

File: src/static/bots/7000103262/index.json [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  "tags": {
+    "zh": ["Ahr999", "information"]
+  },
+  "language": "zh",
+  "languages": ["zh"],
+
+  "developer": {
+    "name": "王卓",
+    "url": "mixin@28865"
```
Comment (lyricat) [inline src/static/bots/7000103262/index.json:22]: if you want to others contact you via mixin, you may replace it with `https://mixin.one/users/44d9717d-8cae-4004-98a1-f9ad544dcfb1`

**Comment by who3m1** [comment]
Sorry, my bad, thanks for correction, I will fix it.

**Pull request closed without merging**